pallets/ddc-validator/src/tests.rs

```diff
@@ -218,3 +225,379 @@ fn it_sets_validation_decision_with_one_validator_in_quorum() {
 		);
 	})
 }
+
+#[test]
+fn send_signal_works_as_expected() {
+	let mut t = new_test_ext();
+
+	let validator_controller = AccountId::from([0xaa; 32]);
+
+	t.execute_with(|| {
+		assert_eq!(DdcValidator::signal(), None);
+		assert_ok!(DdcValidator::send_signal(RuntimeOrigin::signed(validator_controller)));
+		assert_eq!(DdcValidator::signal(), Some(true));
+		DdcValidator::on_initialize(2);
+		System::set_block_number(2);
+		assert_eq!(DdcValidator::signal(), Some(false));
+	})
+}
+
+#[test]
+fn set_validation_decision_works_as_expected() {
+	let mut t = new_test_ext();
+
+	let era_to_validate: EraIndex = 3;
+	let cdn_node_to_validate = AccountId::from([0x1; 32]);
+	let not_a_cdn_node = AccountId::from([0x2; 32]);
+	let validator_1_stash = AccountId::from([
+		0xd2, 0xbf, 0x4b, 0x84, 0x4d, 0xfe, 0xfd, 0x67, 0x72, 0xa8, 0x84, 0x3e, 0x66, 0x9f, 0x94,
+		0x34, 0x08, 0x96, 0x6a, 0x97, 0x7e, 0x3a, 0xe2, 0xaf, 0x1d, 0xd7, 0x8e, 0x0f, 0x55, 0xf4,
+		0xdf, 0x67,
+	]);
```

Comment:
1. We can declare mocks once for all tests. 
2. Can we use something short like `AccountId::from([0x3; 32])`?

Reply:
1. Do you have an example how to get accounts in mock?
2. I think this account corresponds to alice stash, that's why it's so long :)

Reply:
1. I mean we can declare the accounts in the common scope once